test/controllers/sessions_res_controller_test.rb

```diff
@@ -34,7 +34,7 @@ def test_signin_creation_notsuccess
   end
```

Comment:
def test_signin_creation_notsuccess
    value = {identity: "", email: @worker.email, password: @worker.password}
    response = post "/signin_restaurant", value
    assert_equal(response.status,302)
  end


check if value is not there in DB

Reply:
checked for required cases